gcp_variant_transforms/libs/annotation/vep/test_util.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+
+# TODO(bashir2): Create tests with non local inputs as well.
+INPUT_PATTERN = 'some/input/pattern*'
```

Comment:
I think it's odd to include sample patterns here unless it's being used very often. I personally think it's ok to repeat them since it's only two tests for now. If you decide to keep them though, please change to a method like `get_sample_input_data()` that returns both pattern + files and adds some connection between the two. We did a similar thing in [testdata_util.py](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/testing/testdata_util.py#L39).